junit-runtime/src/main/scala/org/junit/After.scala

```diff
@@ -0,0 +1,14 @@
+/*
+ * Ported from https://github.com/junit-team/junit
+ */
+package org.junit
+
+import java.lang.annotation._
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(Array(ElementType.METHOD))
+class After extends scala.annotation.Annotation
+    with java.lang.annotation.Annotation {
+  def annotationType(): Class[Annotation] =
+    classOf[After].asInstanceOf[Class[Annotation]]
```

Comment:
The result type is not correct. It should be `Class[_ <: Annotation]`. Which means the cast is not necessary.
Same for the other annotation classes.